test/game_test.rb

```diff
@@ -75,4 +75,87 @@ def test_DEBUG_board_renders_shot_behavior_for_testing
 
   end
 
+  def test_it_has_a_main_menu
```

Comment:
Confirming I see a test for start --> main menu method

Reply:
Do we need tests for start method by itself or leave_game or any helper methods written in?

Reply:
Yes, there are so many tests I need to do for this.  I'm not doing proper TDD.  But honestly, now that we've merged your awesome fire methods, I really feel like I can dig my teeth into this.  Also, most of the tests that would be written for these are focused on what is being printed to the terminal.  So per Tim's instruction, I am no longer going to test what gets printed to the terminal, but I do need to test functionality.

Reply:
Love it. Yes, I agree. Most needs to be TDD to create, but I was also a little lost on how to TDD when working helper methods. I'm starting to see what is best-practice, but I would LOVE to chat and rework our git add commit flow for any attempt on Iteration 4.